code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -163,123 +119,86 @@
 				try_again = FALSE							// This code will really only come into play on lowpop rounds where getting duplicate targets is more common
 				continue
 		assigned_targets.Add("[kill_objective.target]")
-		add_objective(kill_objective)
-		i += 1
```

Comment:
This will now make it so that if the second target found is a dupe (same as the first target) then the AI will only have 1 kill target and a block target. Low low chance though and only caught once in the current code

Reply:
Yep you're right, and also its such a low chance like you said, I'm wondering if I should even keep the whole "try again" thing around.